### PR TITLE
added new types and corrected transactions for dai to xdai

### PR DESCRIPTION
**Pull request by James-Sangalli**
Added all the types used in ethereum and updated the card to show the correct transaction

### CODE REVIEW

**Comment by hboon** [comment]
Fixes #124

**Comment by James-Sangalli** [comment]
@farrahfazirah your layout is here, the functionality cannot be completed yet however.

**Comment by hboon** [comment]
@James-Sangalli I noticed the .shtml file doesn't validate. I remember Weiwu thinks that developers can write well-formed HTML :P

**Comment by James-Sangalli** [comment]
Completes https://gist.github.com/hboon/7be08b075e3f1b43fe979bc7c0681d64

File: examples/dai/dai.xml [hunk 1]
```diff
@@ -46,9 +52,12 @@
                 </ts:origins>
             </ts:attribute-type>
             <ts:transaction>
-                <ts:ethereum function="dai-bridge" contract="dai-bridge">
+                <!-- convert erc20 DAI to native xDAI -->
+                <!-- simply send dai to 0x4aa42145Aa6Ebf72e164C9bBC74fbD3788045016 -->
+                <ts:ethereum function="transfer" contract="dai">
                     <ts:inputs>
-                        <ts:uint256 ref="amount"/>
+                        <ts:address ref="dst"/>
```
Comment (hboon) [inline examples/dai/dai.xml]: @James-Sangalli Do you mean for "dst" and "wad" to refer to an attribute like "amount"?

Comment (James-Sangalli) [reply examples/dai/dai.xml]: Yes, I am borrowing their naming conventions (which are different for some reason)

**Pull request merged**